src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -97,12 +98,22 @@ public string LiteralPath
         [ValidateNotNullOrEmpty]
         public string SchemaFile { get; set; }
 
+        /// <summary>
+        /// Gets or sets JSON document options.
+        /// </summary>
+        /// <value></value>
+        [Parameter]
+        [ValidateNotNullOrEmpty]
+        [ValidateSet("IgnoreComments", "AllowTrailingCommas")]
+        public string[] Option { get; set; } = Array.Empty<string>();
```

Comment:
Does `ValidateSet` really work for array?

Reply:
Yes, https://learn.microsoft.com/en-us/powershell/scripting/developer/cmdlet/validateset-attribute-declaration?view=powershell-7.4#remarks

Reply:
Since it is an array should it be Option**s**

Reply:
Should we make parameter names plural though? E.g. in other cmdlets we have `-ComputerName` which is singular but can accept an array. I thought powershell recommendation for consistency is to have all array parameters use singular names?

Reply:
I have no strong opinion but:
```powershell
gcm -ParameterName Option

Get-Command: No matching commands include a parameter named 'Option'.  Check the spelling of the parameter name, and then try again. (Parameter 'Option')

gcm -ParameterName Options

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Function        TabExpansion2
Cmdlet          Enter-PSSession                                    7.5.0.0    Microsoft.PowerShell.Core
Cmdlet          Invoke-Command                                     7.5.0.0    Microsoft.PowerShell.Core
Cmdlet          New-PSSession                                      7.5.0.0    Microsoft.PowerShell.Core
Cmdlet          Start-Job                                          7.5.0.0    Microsoft.PowerShell.Core
```